src/haddock/modules/analysis/caprieval/defaults.cfg

```diff
@@ -15,4 +15,7 @@ atoms = ['C','N','CA','O']
 #  this CA will NOT be used for the alignment.
 ignore_missing = true
 sortby = 'score'
-ascending = true
+sort_ascending = true
+# which field should be used to create the ranking
+rankby = 'score'
+rank_ascending = true
```

Comment:
What will happen if both `sort_ascending` and `rank_ascending` are true and `sortby` is set to e.g. i-rmsd while `rankby` is set by score?

Shouldn't we have only `sort_ascending`?

Reply:
> What will happen if both `sort_ascending` and `rank_ascending` are true and `sortby` is set to e.g. i-rmsd while `rankby` is set by score?

This scenario is the example I posted above

Reply:
But aren't `sort_ascending` and `rank_ascending` redundant???  I don't see the need for the `rank_ascending` parameter

Reply:
one defined the direction in which the table will be sorted, the other the direction of the value that we will rank the models;
for sorting: irmsd we'd sort asceding (the lower the best), fnat descending (the higher the best)
for ranking: haddock-score ascending (the lower the best), some other module in which the higher the best = descending

But up to you, I can remove it as well. Rule # 1 !